### ISSUE TITLE
Gamemode History is missing

### ISSUE THREAD

**Issue by riccardocescon** [alexrintt/lichess.dart#25]
Ref: https://lichess.org/api#tag/Users/operation/playerTopNbPerfType

There is a signature but the implementation is missing.
This api should give a user history for a requested gamemode 'blitz', 'classic'....

**Comment by alexrintt** [comment]
> This api should give a user history for a requested gamemode

The endpoint you mentioned has nothing to do with user history.

**Comment by alexrintt** [comment]
[playerTopNbPerfType](https://lichess.org/api#tag/Users/operation/playerTopNbPerfType) was implemented, but i am not sure if it covers your use-case (show the match history of a given user).

**Comment by alexrintt** [comment]
Ok, to list the user most recent matches first, we probably need to use this endpoint: https://lichess.org/api#tag/Games/operation/apiUserCurrentGame.

**Comment by riccardocescon** [comment]
Yes my bad, noticed It Yesterday, i started using this endpoint and forgot to update this issue
